components/ui/svg.tsx

```diff
@@ -35,6 +38,45 @@ export class SvgImage {
     const {width, height} = props;
     return <Logo width={width} height={height} />;
   }
+  static home(focused: boolean) {
+    //TODO: Here tab icons names should be same with key "name" in main.ts
```

Comment:
can you please let me know why is this TODO, shouldn't it a note or simple comments

Reply:
It's a note that will guide the names of icons in svg.tsx & the bottom tab paramslist should be the same. Otherwise, the implements regarding icons will not be implemented for tab icons.